common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -190,6 +195,15 @@ private void maybeStartReporting() {
         if (reportingThread != null) {
             return;
         }
+        if (!firstReportHasRun) {
+            buildSpan("lightstep.tracer_create")
+                    .ignoreActiveSpan()
+                    .withTag("lightstep.meta_event", true)
```

Comment:
My two cents: could we put these tag names somewhere, i.e. `Tags` or `Constants`? We don't need to use constants always, but in this case it *feels* right ;)

Reply:
Yeah, I'm thinking about going through and doing all of these as constants. Will probably go with that.